_reports/chay2199.md

```diff
@@ -0,0 +1,331 @@
+---
+layout: report
+student: Chaitanya Tyagi
+organisation: coala
+organisation_link : https://coala.io
+project: Result Reporter Tool
+project_link: https://summerofcode.withgoogle.com/projects/?sp-search=chaitanya#6527640138153984
+tarball: https://github.com/chay2199/GSoCReport/raw/master/chay2199_coala.tar.xz
+mentor: >
+ [Boxuan Li](https://github.com/li-boxuan)
+phase:
+ - Bonding : https://gitlab.com/coala/GSoC/gsoc-2019/-/milestones/6?title=BONDING_RESULT_REPORTER_TOOL
+ - Phase 1 : https://gitlab.com/coala/GSoC/gsoc-2019/-/milestones/7?title=PHASE_1_RESULT_REPORTER_TOOL
+ - Phase 2 : https://gitlab.com/coala/GSoC/gsoc-2019/-/milestones/8?title=PHASE_2_RESULT_REPORTER_TOOL
+ - Phase 3 : https://gitlab.com/coala/GSoC/gsoc-2019/-/milestones/9?title=PHASE_3_RESULT_REPORTER_TOOL
+bio: >
+ I'm a third year student (expected graduation date: May 2021) of Computer Science at Birla Institute of Technology, Mesra. I participated in GSoC and worked with [coala](http://coala.io) on building a Result Reporter Tool, which loads data from coala-json output and converts it to test result report formats like JUnit, TAP and Checkstyle. [coala-json](https://pypi.org/project/coala-json/) is a python package now, and thus it can be extended to any other project which wants to generate test reports using their json output simply by changing the loader file.
```

Comment:
This is a super long line; consider breaking it into more lines.

Use https for coala.io link